src/controllers/trading.status.controller.ts

```diff
@@ -1,6 +1,13 @@
 import { Request, Response } from "express";
 import { Templates } from "../types/template.paths";
+import { TASK_LIST_PATH, urlParams } from "../types/page.urls";
 
 export const get = (req: Request, res: Response) => {
   return res.render(Templates.TRADING_STATUS);
 };
+
+export const post = (req: Request, res: Response) => {
+  const companyNumber =  req.params[urlParams.PARAM_COMPANY_NUMBER];
+  const url = TASK_LIST_PATH.replace(":companyNumber", companyNumber);
```

Comment:
a minor thing, the ":companyNumber" can use the param constant eg
``:${urlParams.PARAM_COMPANY_NUMBER}``

Reply:
> a minor thing, the ":companyNumber" can use the param constant eg
> `:${urlParams.PARAM_COMPANY_NUMBER}`

there are supposed to be 'backtick' quotes around my example but github interprets them as formatting, but you should be able to se what I mean.